app/controllers/application_controller.rb

```diff
@@ -1,7 +1,13 @@
 class ApplicationController < ActionController::Base
   helper_method :current_user
+  helper_method :find_artist_by_id
+
 
   def current_user
     @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
   end
+
+  def find_artist_by_id(id)
```

Comment:
Excellent idea to pull the need for User.find(id) out of the controller with this helper method! Slick.